manifests/base/kustomization.yaml

```diff
@@ -671,7 +671,6 @@ replacements:
     targets:
       - fieldPaths:
           - spec.tags.0.from.name
-          - spec.tags.0.annotations.[opendatahub.io/runtime-image-metadata].0.metadata.image_name
```

Comment:
* https://github.com/opendatahub-io/kubeflow/pull/603

yes, got it, we're getting quite far from the original intent/title of the PR, though

> Standardize the image name with image convention

edit: I now see why it's necessary to do it all in the same PR, so ok

Reply:
I wasnt able to find a way to separate it out.
perhaps we can consider it for this one time.

Reply:
Its fine, the changes do belong together. And hopefully this PR will not drag too long, because there's actually urgency to getting all this sorted out, so it's not a problem 🙏🏽